shanoir-ng-front/src/app/shared/components/subject-study-list/subject-study-list.component.ts

```diff
@@ -185,11 +184,11 @@ export class SubjectStudyListComponent extends AbstractInput<SubjectStudy[]> imp
         if (this.selectableList && this.model) {
             if (this.compMode == 'study') {
                 for (const option of this.optionList) {
-                    if(this.model.find(subStu => subStu.subject.id == option.value.id)) option.disabled = true;
+                    if(this.model.find(subject => subject.id == option.value.id)) option.disabled = true;
                 }
             } else if (this.compMode == 'subject') {
                 for (const option of this.optionList) {
-                    if(this.model.find(subStu => subStu.study.id == option.value.id)) option.disabled = true;
+                    if(this.model.find(subject => subject.study.id == option.value.id)) option.disabled = true;
                 }
             }
         }
```

Comment:
Remove completely compMode since now the component is only used in the 'sudy' mode.

This should remove a lot of code.